common/logisticspipes/proxy/specialinventoryhandler/AEInterfaceInventoryHandler.java

```diff
@@ -62,7 +66,13 @@ public AEInterfaceInventoryHandler() {
 
 	@Override
 	public boolean isType(TileEntity tile, EnumFacing dir) {
-		return tile instanceof IGridHost && tile.hasCapability(LPStorageMonitorableAccessor.STORAGE_MONITORABLE_ACCESSOR_CAPABILITY, dir);
+		if (tile instanceof IGridHost && tile.hasCapability(LPStorageMonitorableAccessor.STORAGE_MONITORABLE_ACCESSOR_CAPABILITY, dir)) {
+			IGridNode tempNode = ((IGridHost) tile).getGridNode(AEPartLocation.fromFacing(dir));    //for some reason when AE loads (5 ticks) this is null
+			if (tempNode == null)																	//should not be a problem because interfaces are still treated like IInvertoryHandler
```

Comment:
That comment shouldn't fly away

Reply:
In idea and notepad++ looks perfectly fine. Anyway i'm going to remove it.